e2e/external_dep/WORKSPACE

```diff
@@ -0,0 +1,29 @@
+workspace(name = "lib_wksp")
```

Comment:
Oh shoot. git is going to get confused between the file `WORKSPACE` and the folder `workspace`.

One of them needs to be renamed to something else.

Reply:
Grr, case-insensitive file names strike again! Update the `workspace/` directory to `app/`, which is probably a more accurate name for its intended abstraction.